spec/system/custom/admin/legislation/processes_spec.rb

```diff
@@ -104,6 +104,11 @@
 
       expect(page).to have_content "New title"
       expect(page).to have_content "New description"
+
```

Comment:
Since we now don't expect the prompt to show on the debates phase, maybe we can make it explicit :thinking:.

```suggestion
      expect(page).not_to have_content "NEW PROMPT?"
```